### ISSUE TITLE
Bug: CIMSolidStroke is not drawn properly with CIMTextSymbol

### ISSUE THREAD

**Issue by kawamikasa** [Esri/feedback-js-api-next#69]
![CIMSolidStroke-bug](https://user-images.githubusercontent.com/70646069/94566836-1b6fcc80-026b-11eb-98a3-403d1c0a11a9.PNG)

```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta
      name="viewport"
      content="initial-scale=1,maximum-scale=1,user-scalable=no"
    />
    <title>CIMSymbol | Sample | ArcGIS API for JavaScript 4.16</title>
    <style>
      html,
      body,
      #viewDiv {
        padding: 0;
        margin: 0;
        height: 100%;
        width: 100%;
      }
    </style>

    <link
      rel="stylesheet"
      href="https://js.arcgis.com/4.16/esri/themes/light/main.css"
    />
    <script src="https://js.arcgis.com/4.16/"></script>

    <script>
      require([
        "esri/views/MapView",
        "esri/Map",
        "esri/Graphic",
        "esri/layers/GraphicsLayer"
      ], function (
        MapView,
        Map,
        Graphic,
        GraphicsLayer
      ) {
        const graphic = new Graphic({
            geometry: {
              type: "point",
              latitude: 31.19,
              longitude: 122.12
            },
            symbol: getSymbol()
        });
        
        const graphicsLayer = new GraphicsLayer({
          graphics: [graphic]
        });

        const map = new Map({
          basemap: "gray-vector",
          layers: [graphicsLayer]
        });

        const view = new MapView({
          container: "viewDiv",
          map: map,
          center: [106.79, 38.23],
          zoom: 3
        });

        function getSymbol() {
          return {
            type: 'cim',
            data: {
              type: 'CIMSymbolReference',
              symbol: {
                type: 'CIMPointSymbol',
                symbolLayers: [
                  {
                    type: 'CIMVectorMarker',
                    enable: true,
                    size: 38,
                    anchorPoint: { x: 0, y: 0 },
                    anchorPointUnits: 'Relative',
                    frame: { xmin: -164, ymin: -33, xmax: 164, ymax: 33 },
                    markerGraphics: [
                      {
                        type: 'CIMMarkerGraphic',
                        geometry: { x: 0, y: 0 },
                        symbol: {
                          type: 'CIMTextSymbol',
                          fontFamilyName: 'Arial Unicode MS',
                          fontStyleName: 'Normal',
                          height: 15,
                          horizontalAlignment: 'Center',
                          offsetX: 0,
                          offsetY: 0,
                          symbol: {
                            type: 'CIMPolygonSymbol',
                            symbolLayers: [
                              { 
                                type: 'CIMSolidFill', 
                                enable: true, 
                                color: [220, 20, 60, 255] 
                              }
                            ],
                          },
                          verticalAlignment: 'Center',
                        },
                        textString: '45654\nMQELgMf9XYAs8RrIIxbGptRPeF\nsdqsdse',
                      },
                    ],
                    scaleSymbolsProportionally: true,
                    respectFrame: true,
                  },
                  {
                    type: 'CIMVectorMarker',
                    enable: true,
                    size: 38,
                    anchorPoint: { x: 0, y: 0 },
                    anchorPointUnits: 'Relative',
                    frame: { xmin: -164, ymin: -33, xmax: 164, ymax: 33 },
                    markerGraphics: [
                      {
                        type: 'CIMMarkerGraphic',
                        geometry: {
                          rings: [
                            [
                              [-162, 31],
                              [162, 31],
                              [162, -31],
                              [-162, -31],
                              [-162, 31],
                            ],
                          ],
                        },
                        symbol: {
                          type: 'CIMPolygonSymbol',
                          symbolLayers: [
                            { 
                              type: 'CIMSolidStroke', 
                              enable: true, 
                              width: 2, 
                              color: [0, 0, 0, 255] 
                            }
                          ],
                        },
                      },
                    ],
                    scaleSymbolsProportionally: true,
                    respectFrame: true,
                  },
                ],
              },
            },
          }
        }
      });
    </script>
  </head>

  <body>
    <div id="viewDiv">
    </div>
  </body>
</html>
```

**Comment by annelfitz** [comment]
Hi @kawamikasa - thanks for reporting this. I'd just like to remind you that this repo is for the unreleased "next" version. The best way to log issues with released software is to contact Esri Support. 

With this issue specifically, it is a known issue and we are working on a fix. Thanks!

**Comment by annelfitz** [comment]
Hi @kawamikasa - this will be fixed in the 4.21 release: https://codepen.io/annefitz/pen/NWNVjRZ
